api/v1alpha7/openstackmachine_types.go

```diff
@@ -43,7 +43,11 @@ type OpenStackMachineSpec struct {
 	CloudName string `json:"cloudName"`
 
 	// The flavor reference for the flavor for your server instance.
-	Flavor string `json:"flavor"`
+	Flavor string `json:"flavor,omitempty"`
```

Comment:
We wouldn't normally change old API versions, but I'm inclined to let this slide, tbh.